wrappers/csharp/src/Frame.cs

```diff
@@ -195,6 +195,60 @@ public float GetDistance(int x, int y)
         }
     }
 
+    public class Points : Frame
+    {
+        [System.Runtime.InteropServices.StructLayout(System.Runtime.InteropServices.LayoutKind.Sequential)]
+        public struct Vertex
```

Comment:
Is it possible to use `UnityEngine.Vector3`?

Reply:
Since this is Csharp wrapper, I thought it was not appropriate to use namespace UnityEngine.
Also, since Vector 3 contains extra properties other than coordinates, it is not appropriate.

Reply:
Ohh I must have floated away with this one, please ignore :D